spec/introduction.md

```diff
@@ -21,17 +21,21 @@ using System;
 
 class Hello
 {
-    static void Main() {
+    static void Main()
+    {
         Console.WriteLine("Hello, World");
     }
 }
 ```
 
-C# source files typically have the file extension `.cs`. Assuming that the "Hello, World" program is stored in the file `hello.cs`, the program can be compiled with the Microsoft C# compiler using the command line
+C# source files typically have the file extension *.cs*. Assuming that the "Hello, World" program is stored in the file *hello.cs*, the program can be compiled with the Microsoft C# compiler using the command line:
+
 ```console
 csc hello.cs
 ```
-which produces an executable assembly named `hello.exe`. The output produced by this application when it is run is
+
+which produces an executable assembly named *hello.exe*. The output produced by this application when it is run is:
```

Comment:
Switching from `code` style to *italic* for all these file names (and parts of file names) seems like a significant backward step to me.

Reply:
@theunrepentantgeek, that's the current style in the dotnet/docs

Reply:
Fair enough.